README.md

```diff
@@ -1 +1,100 @@
-# perl-App-Config
\ No newline at end of file
+# NAME
+
+App::Config - The great new App::Config!
+
+# VERSION
+
+Version 0.01
+
+# SYNOPSIS
+
+    my $app_config = App::Config->new;
+
+# DESCRIPTION
+
+This module parses configuration files and provides interface to access
+configuration information.
+
+# FILE FORMAT
+
+The configuration file is a YAML file. Here is an example:
+
+    system:
+      description: "Various parameters determining core application functionality"
+      isa: section
+      contains:
+        email:
+          description: "Dummy email address to indicate email sent from our system"
+          isa: Str
+          default: "dummy@binary.com"
```

Comment:
don't use binary.com; this is an open-source module which should have nothing to do with binary.com